snap/snapcraft.yaml

```diff
@@ -0,0 +1,46 @@
+name: foundationdb
+version: 6.0.18
+summary:  An open source, distributed, transactional key-value store.
+description:| 
+A multi-model NoSQL database.
+
+grade: stable
+confinement: classic
+base: core18
+
+architectures:
+  - build-on: amd64
+
+apps:
+  foundationdb:
+    command: bin/desktop-launch $SNAP/bin/FoundationDB
```

Comment:
How does this work?  Where is the definition of what desktop-launch does to get a running fdbmonitor+fdbserver?  How is a cluster file generated?